lessons/2_more_strings_and_functions/lesson.py

```diff
@@ -372,25 +419,85 @@ def inner_func():
 # 1. Write a function that takes a string, sorts the characters in the string, and returns a string
 #    with the individual characters sorted. Name this function sort_chars.
 
-# 2. Write a function that takes a string and returns the value of the character at the index which is half of the
-#    total length of the string  if the string is even, the value of the penultimate character if the string is odd, 
+def sort_chars(string):
+    string_sorted = sorted(string)
+    string_sorted = "".join(string_sorted)
+    return string_sorted
```

Comment:
This is correct, but there is technically no need to define string_sorted since you don't need to maintain the list that sorted() returns. This code can be reduced to one line:

```
def sort_chars(string):
    return "".join(sorted(string)
```